rt_utils/rtstruct_builder.py

```diff
@@ -79,7 +79,9 @@ def validate_contour_image_in_series_data(
                 return
 
         # ReferencedSOPInstanceUID is NOT available
-        raise Exception(
-            f"Loaded RTStruct references image(s) that are not contained in input series data. "
-            + f"Problematic image has SOP Instance Id: {contour_image.ReferencedSOPInstanceUID}"
-        )
+        msg = f"Loaded RTStruct references image(s) that are not contained in input series data. "
```

Comment:
Build is failing due to indentation. Maybe replace with

```
        msg = f"Loaded RTStruct references image(s) that are not contained in input series data. " + \
            f"Problematic image has SOP Instance Id: {contour_image.ReferencedSOPInstanceUID}"
```

Reply:
Done [9ca7a20](https://github.com/qurit/rt-utils/pull/64/commits/9ca7a20e00e18d36c150f2ca677756d61adeea6e)